src/pkg/packaging-tools/framework.dependency.targets

```diff
@@ -143,11 +145,11 @@
     </PropertyGroup>
 
     <GenerateFileVersionProps Files="@(SharedFrameworkRuntimeFiles)"
-                              PackageId="$(FrameworkPackageName)"
+                              PackageId="__PackageId__"
```

Comment:
Why these items are now hardcoded here?

Reply:
They aren't, this is a templating token that's replaced in the pkgproj. The commit + commit message might shed some light: https://github.com/dotnet/core-setup/pull/7259/commits/cc3c81b6caec5947f03f02ecb306610236e82fee.

Reply:
Doh!  If I would have read a little further, I would see where the replacement happens.  Thanks.

Reply:
Would actually be a good place for a comment, on second thought. It's only in context in this PR, not when people read the file itself later.